yt/visualization/particle_plots.py

```diff
@@ -270,28 +386,58 @@ def __init__(
             self._use_cbar = False
             splat_color = color
 
-        depth = ds.coordinates.sanitize_depth(depth)
+        if isinstance(normal, (int, str)):
```

Comment:
at this point, `normal` was already sanitized and can only be of type `str` or `np.ndarray` (unless `NormalPlot.sanitize_normal_vector`'s annotations are wrong ?). Something is inconsistent here.

I think the class hierarchy for `ParticleProjectionPlot` should follow the same design pattern as `SlicePlot` and `ProjectionPlot`.

Reply:
I am not sure why offhand, but the code didn't work correctly without this last week. I'll have to look but I am not sure how much time I am going to have in the rest of this week.